docs/source/intro/tutorial.md

```diff
@@ -501,22 +501,42 @@ class MultiplyAddWorkChain(WorkChain):
 ```
 
 :::{note}
-Besides WorkChain's, workflows can also be implemented as *work functions*.
+Besides WorkChain's, workflows can also be implemented as [*work functions*](https://aiida.readthedocs.io/projects/aiida-core/en/latest/topics/workflows/concepts.html).
 These are ideal for workflows that are not very computationally intensive and can be easily implemented in a Python function.
 :::
 
 Let's run the `WorkChain` above!
+
+:::{note}
+To simplify the basic tutorial, we have left out the steps to register the `MultiplyAddWorkChain` as a plugin. This allows us to access it through the [*entry point*](https://aiida.readthedocs.io/projects/aiida-core/en/latest/topics/plugins.html#topics-plugins-entrypoints) named `core.arithmetic.multiply_add`.
+
+```console
+$ verdi plugin list
+```
+to show the available pulgins and their entry points.
```

Comment:
this command only shows the entry points groups, it'll be better to elaborate a bit and provide an example of `verdi plugin list aiida.workflows core.arithmetic.multiply_add`.